source/edge-server/configure.txt

```diff
@@ -386,3 +386,80 @@ server, as well as errors that may have occurred.
             "Item": "truepredicate"
          }
       }
+
+.. _upgrade-edge-server-version:
+
+Upgrade the Edge Server Version
+-------------------------------
+
+As the Edge Server evolves, you may want to upgrade your Edge Server 
+version to take advantage of new functionality.
+
+Prerequisites
+~~~~~~~~~~~~~
+
+Upgrading the Edge Server version requires the host machine to have 
+`jq <https://jqlang.github.io/jq/>`__ installed. If you do not already
+have ``jq`` installed, running an Edge Server upgrade command tells you 
+to install ``jq``.
+
+.. _edge-server-upgrade-commands:
+
+Upgrade Edge Server Commands
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can upgrade the Edge Server version using one of three commands:
+
+- ``make upgrade-major``: Upgrade to the latest major version of Edge Server.
+- ``make upgrade-minor``: Upgrade to the latest minor version of Edge Server.
+- ``make upgrade-force``: Upgrade to the latest version of Edge Server, 
+  regardless of major or minor version status.
+
+Upgrade Edge Server Procedure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. procedure::
+
+   .. step:: Stop the Edge Server
+
+      From the ``edge_server`` directory, run the command to stop the Edge 
```

Comment:
```suggestion
      From the ``edge_server`` directory, run the following command to stop the Edge 
```

Reply:
Stopping is not strictly required, the upgrade script will stop it for you. So you can omit this step